### ISSUE TITLE
[Package Fix Request]: VideoLAN.VLC

### ISSUE THREAD

**Issue by thedoggybrad** [microsoft/winget-pkgs#58612]
### Package Requested

- [X] I would like someone else to build the manifest.
- [ ] I would like help so I can submit the manifest.
- [ ] I have performed a search and couldn't find this package.
- [ ] I have checked that the installer for this package is not a standalone or portable executable or a .zip file
- [ ] I think there is a new version available and I have provided the URL.

### Please provide the following information

This is a duplicate but I want to provide my own suggestions.

This is a good recommendation to fix this:
Remove both VLC version 3.0.17.4 and VLC version 3.0.17

Then make a manifest of VLC version 3.0.17.4 but label it as 3.0.17.

To sum up,
VLC 3.0.17 - Remove
VLC 3.0.17.4 - Rename to 3.0.17

Hope this fixes the issue. I am not an expert so I let others do this.

**Comment by vedantmgoyal2009** [comment]
Duplicate of https://github.com/microsoft/winget-pkgs/issues/58397

**Comment by thedoggybrad** [comment]
Can anyone attempt to do my sugestions?

**Comment by Trenly** [comment]
Your suggestions won’t work because 3.0.17.4 is a security fix, and so we cannot backversion it to 3.0.17. Therefore, the publisher must fix their MSI installer

Duplicate of #58397
